std_msgs/Header header
float64 thrust # total thrust adjustment for height control
geometry_msgs/Vector3 omega # angular rate in body frame
bool is_last_command
